test/functional/services/lib/web_element_wrapper/web_element_wrapper.ts

```diff
@@ -17,45 +17,73 @@
  * under the License.
  */
 
-import { scrollIntoViewIfNecessary } from './scroll_into_view_if_necessary';
 import { delay } from 'bluebird';
+import { WebElement, WebDriver, By, IKey, until } from 'selenium-webdriver';
+// @ts-ignore not supported yet
 import { PNG } from 'pngjs';
+// @ts-ignore not supported yet
 import cheerio from 'cheerio';
 import testSubjSelector from '@kbn/test-subj-selector';
+import { ToolingLog } from '@kbn/dev-utils';
+// @ts-ignore not supported yet
+import { scrollIntoViewIfNecessary } from './scroll_into_view_if_necessary';
+
+interface Driver {
+  driver: WebDriver;
+  By: typeof By;
+  Key: IKey;
+  until: typeof until;
+  LegacyActionSequence: any;
+}
+
+interface TypeOptions {
+  charByChar: boolean;
+}
 
 export class WebElementWrapper {
-  constructor(webElement, webDriver, timeout, fixedHeaderHeight, log) {
+  private By: typeof By = this.webDriver.By;
+  private Keys: IKey = this.webDriver.Key;
+  private driver: WebDriver = this.webDriver.driver;
+  public _webElement: WebElement = this.webElement as WebElement;
+  public LegacyAction: any = this.webDriver.LegacyActionSequence;
+
+  constructor(
+    private webElement: WebElementWrapper | WebElement,
```

Comment:
Seems like this doesn't need to be a property, and we should rename the `_webElement` property to use the unprefixed name.

Reply:
unfourchantly, we have a lot of syntax like `element._webElement` e.g.. Find methods `allDescendantDisplayedByCssSelector`, `displayedByLinkText`, etc . it made to avoiding breaking changes

Reply:
ugh, alright. Feel free to make broad improvements like that as a subsequent PR if you like.